.golangci.yml

```diff
@@ -333,3 +333,10 @@ issues:
     - path: mip/options\.go
       linters:
         - lll
+    # Although the name stutters, we need to call it mip.MipOptions because the
+    # mip package can also handle linear programs, so potentially we may have
+    # mip.LpOptions.
```

Comment:
If only we would have named it _linear_ or similar. :sweat_smile: 
For the technically correct: this of course clashes with any quadratic stuff.
Good point though!